### PR TITLE
More Keycloak 9 related updates

### LINKED ISSUE #467: Update the realm file to the Keycloak 9 compliant version in OIDC quickstarts

### PR DESCRIPTION
**Pull request by sberyozkin**
I'll follow up with a minor update for the oidc quickstart doc too.
Fixes #467 (with Pedro already setting the the version/realm for the mutitenancy quickstart)

### CODE REVIEW

File: security-openid-connect-web-authentication-quickstart/pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
     <properties>
         <quarkus.version>999-SNAPSHOT</quarkus.version>
-        <keycloak.version>7.0.1</keycloak.version>
+        <keycloak.version>9.0.0</keycloak.version>
```
Comment (gsmet) [inline security-openid-connect-web-authentication-quickstart/pom.xml]: Is this property used, I don't see it used?

Comment (sberyozkin) [reply security-openid-connect-web-authentication-quickstart/pom.xml]: @gsmet sorry, indeed, it is in the code, let me update a bit

Comment (sberyozkin) [reply security-openid-connect-web-authentication-quickstart/pom.xml]: @gsmet Update the code in two quaickstarts to check the system property

**Comment by gsmet** [comment]
@sberyozkin this needs a rebase.

**Comment by sberyozkin** [comment]
Hi @gsmet Done :-)

**Comment by gsmet** [comment]
I rebased and force-pushed, let's wait for CI.

**Pull request merged**